### PR TITLE
Release documentation for 2185 build

### PR DESCRIPTION
**Pull request by shramroy**

### CODE REVIEW

File: docs/news.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # News
 
+* **8th September, 2021**: Ixia-c version 0.0.1 (build 2185) released.  This conforms to **[v0.5.4](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.5.4/openapi.yaml)** of the Open Traffic Generator Models specification.
+
+  * This build contains support for updating flow rate without disruption of transmit state. [Read more](releases.md)
```
Comment (ashutshkumr) [inline docs/news.md:5]: without disrupting transmit state

File: docs/releases.md [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
+|-------------------------------|---------------|
+| Open Traffic Generator API    | [0.5.4](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.5.4/openapi.yaml)         |
+| snappi                        | [0.5.3](https://pypi.org/project/snappi/0.5.3)        |
+| ixia-c-controller             | [0.0.1-2185](https://hub.docker.com/r/ixiacom/ixia-c-controller/tags)    |
+| ixia-c-traffic-engine         | [1.4.0.11](https://hub.docker.com/r/ixiacom/ixia-c-traffic-engine/tags)       |
+| ixia-c-app-usage-reporter     | [0.0.1-37](https://hub.docker.com/r/ixiacom/ixia-c-app-usage-reporter/tags)      |
+| ixia-c-protocol-engine        | 1.00.0.56     | 
+| otg-gnmi-server               | [0.5.2](https://hub.docker.com/r/otgservices/otg-gnmi-server/tags)         |
+| otg-grpc-server               | [0.5.3](https://hub.docker.com/r/otgservices/otg-grpc-server/tags)         |
+| operator                      | 0.0.35        |
```
Comment (ashutshkumr) [inline docs/releases.md:22]: remove operator

File: docs/news.md (hunk 1 above)
Comment (shramroy) [reply docs/news.md]: changed

File: docs/releases.md (hunk 2 above)
Comment (shramroy) [reply docs/releases.md]: updated

File: docs/releases.md [hunk 3]
```diff
@@ -1,6 +1,37 @@
...
+
+
+#### New Feature(s)
+
+* Updating flow rate without disruption of transmit state is now supported. Rate of multiple flows can be updated simultaneously through `update_flows` api without stopping the traffic. 
+
+#### Known Issues
+
+* The metric `loss` in flow metrics is currently not supported.
```
Comment (ashutshkumr) [inline docs/releases.md:31]: do we need to mention this limitation every release ? @ankur-sheth

File: docs/releases.md [hunk 4]
```diff
@@ -1,6 +1,37 @@
...
+| ixia-c-app-usage-reporter     | [0.0.1-37](https://hub.docker.com/r/ixiacom/ixia-c-app-usage-reporter/tags)      |
+| ixia-c-protocol-engine        | 1.00.0.56     | 
+| otg-gnmi-server               | [0.5.2](https://hub.docker.com/r/otgservices/otg-gnmi-server/tags)         |
+| otg-grpc-server               | [0.5.3](https://hub.docker.com/r/otgservices/otg-grpc-server/tags)         |
+| operator                      | 0.0.35        |
+
+
+#### New Feature(s)
+
+* Updating flow rate without disruption of transmit state is now supported. Rate of multiple flows can be updated simultaneously through `update_flows` api without stopping the traffic.
```
Comment (ashutshkumr) [inline docs/releases.md]: without disrupting transmit state

Comment (shramroy) [reply docs/releases.md]: I changed this one also. May be old desc is reflecting here.

**Review by ashutshkumr — CHANGES_REQUESTED** [review]
Please also address following:
- Wherever we're initializing `api()`, we should pass `verify=False` to turn off insecure certificate warning. This will allow us to run tests without passing `-W` switch. e.g. `python -W ignore script.py` (such commands in doc should also be modified)
- in hello-snappi.md, document what verify=False does
- add documentation for `add()`, `append()`, `clear()` and `remove()`

File: docs/releases.md [hunk 5]
```diff
@@ -1,6 +1,37 @@
...
+|-------------------------------|---------------|
+| Open Traffic Generator API    | [0.5.4](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.5.4/openapi.yaml)         |
+| snappi                        | [0.5.3](https://pypi.org/project/snappi/0.5.3)        |
+| ixia-c-controller             | [0.0.1-2185](https://hub.docker.com/r/ixiacom/ixia-c-controller/tags)    |
+| ixia-c-traffic-engine         | [1.4.0.11](https://hub.docker.com/r/ixiacom/ixia-c-traffic-engine/tags)       |
+| ixia-c-app-usage-reporter     | [0.0.1-37](https://hub.docker.com/r/ixiacom/ixia-c-app-usage-reporter/tags)      |
+| ixia-c-protocol-engine        | 1.00.0.56     | 
+| otg-gnmi-server               | [0.5.2](https://hub.docker.com/r/otgservices/otg-gnmi-server/tags)         |
+| otg-grpc-server               | [0.5.3](https://hub.docker.com/r/otgservices/otg-grpc-server/tags)         |
+| Resolve operator              | 0.0.35        |
```
Comment (ashutshkumr) [inline docs/releases.md:22]: I meant remove mention of operator altogether.

Comment (shramroy) [reply docs/releases.md:22]: okay

**Comment by shramroy** [comment]
> Please also address following:
> 
> * Wherever we're initializing `api()`, we should pass `verify=False` to turn off insecure certificate warning. This will allow us to run tests without passing `-W` switch. e.g. `python -W ignore script.py` (such commands in doc should also be modified)
> * in hello-snappi.md, document what verify=False does
> * add documentation for `add()`, `append()`, `clear()` and `remove()`

Instead of passing "False" every time will it be better if we change the default value to "false" ?

**Review by ashutshkumr — APPROVED** [review]

**Comment by ankur-sheth** [comment]
Good work getting this release out!  And good additions to hello-snappi.md!

**Pull request merged**